src/jobs/jobs.controller.ts

```diff
@@ -670,9 +764,7 @@ export class JobsController {
    * Delete a job
    */
   @UseGuards(PoliciesGuard)
-  @CheckPolicies(
-    (ability: AppAbility) => ability.can(AuthOp.Delete, JobClass), // TBD
-  )
+  @CheckPolicies((ability: AppAbility) => ability.can(AuthOp.JobDelete, JobClass)&& ability.can(AuthOp.JobDeleteAny, JobClass) )
```

Comment:
Need some more details on this. Do we need to check whether the specific used has the rights to delete a specific job? Is this check enough?

When logged in as an admin, I tried to delete a job that I created but got error `403: forbidden resource`

Reply:
Yes, If you look in the [docs](https://scicatproject.github.io/documentation/Development/v4.x/backend/authorization/authorization_jobs.html) (first table) admin shouldn't have rights to delete jobs, unless he/she is in the DELETE_JOB_GROUPS as defined in env file. I put archiveManager similar to datasets.